### DISCUSSION TITLE
about opentcs-kernel extension-statistics

### DISCUSSION THREAD

**Discussion by SHMay100G** [openTCS/opentcs#107] [Q&A]
Hello, I am a beginner in OpenTCS. I am able to run the project successfully, but I do not see any relevant content about opentcs-kernel extension-statistics in the project. How can I obtain the entire result data after running the project?

**Comment by swltr** [comment]
In the Operations Desk application, you can open a statistics plugin panel via the main menu. Using that panel, you can open the file written by the statistics kernel extension. (The files are output in the kernel application's `log/statistics-extension` subdirectory.) Note that the file is intended to be opened offline / when the kernel is not running, so you probably need to copy it somewhere else, first. Also note that this kernel extension is merely meant as a way to demonstrate how you could hook a statistics component into the kernel. It probably does not provide the functionality that you might expect in a production environment.
